_docs/manage/policies.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: guide
+title: StorageOS Docs - Policies
+anchor: manage
+module: manage/policies
+---
+
+# Policies
+
+Policies are a simple Attribute-Based Access Control records that are used to permission users (or groups) to namespaces.
+
+>**Note**: On initial startup there is only the storageos user and no policies are in place. If no policies exist, all users can access all namespaces.
+
+## Format
+
+The format for policy rules is similar to that of Kubernetes' ABAC syntax, defining acceptable actions within the system. If any of the policy entries match the
+action is allowed.
+
+>**Note**: Admin users are treated as super-users which can perform any action within the system, regardless of policies set.
+
+```json
+{"spec":{"user": "bob", "namespace": "legal"}}
+{"spec":{"group": "compliance", "namespace": "legal"}}
+```
+
+The rules above will allow any user with the username bob, or in the group compliance, to access the legal namespace.
+
+## CLI
+
+### Adding policies
+Policies can be added via the CLI application using the `storageos policy create command. New policies can be supplied to this command with n-many positional arguments, a set of files or by reading from stdin. The positional arguments are expected to be in the form of either a single policy object or a json list of policy objects. The file input is expected to be in json line format (one object per line, the same as kubernetes). Reading from stdin will accept either json line format, or json objects/lists.
+
+#### Adding using positional args
```

Comment:
In general avoid going to 4 level subheadings. I would use # for the page title, and # and ##; more than that gets confusing.